lama-server.py

```diff
@@ -1,10 +1,8 @@
-from game import GameMaster
-from twisted.internet import reactor, endpoints
-from twisted.web import server
+from game import TestMaster
 import sys
 
 if __name__ == "__main__":
-    g = GameMaster()
-    endpoint = endpoints.TCP4ServerEndpoint(reactor, 1144)
-    endpoint.listen(server.Site(g))
-    reactor.run()
+ 
+    with TestMaster() as test:
```

Comment:
You can just make a new file. You don't have to delete current functionality to add more functionality.

Reply:
Oh okay, will do